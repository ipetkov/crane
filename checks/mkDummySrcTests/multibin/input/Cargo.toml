[package]
name = "multibin" # Loadbearing name matching bin
version = "0.1.0"
edition = "2021"

[features]
default = ["some"]
some = []

[[bin]]
name = "alt"
path = "src/bin/hello.rs"

[[bin]]
name = "bye"
required-features = ["some"]

[[bin]]
name = "multibin" # Name of the package
