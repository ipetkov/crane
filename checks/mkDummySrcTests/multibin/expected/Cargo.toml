[[bin]]
name = "alt"
path = "src/bin/hello.rs"

[[bin]]
name = "bye"

[[bin]]
name = "multibin"

[features]
default = ["some"]
some = []

[package]
build = "cranespecific-dummy.rs"
edition = "2021"
name = "multibin"
version = "0.1.0"
